python/cuda_cooperative/cuda/cooperative/experimental/block/_block_load_store.py

```diff
@@ -74,6 +124,49 @@ def load(dtype, threads_in_block, items_per_thread=1, algorithm="direct"):
 
 
 def store(dtype, threads_in_block, items_per_thread=1, algorithm="direct"):
+    """Creates an operation that performs a block-wide store.
+
+    Returns a callable object that can be linked to and invoked from device code. It can be
+    invoked with the following signatures:
+
+    - `(dest: numba.types.Array, src: numba.types.Array) -> dtype`: Each thread stores
```

Comment:
important: same here
```suggestion
    - `(dest: numba.types.Array, src: numba.types.Array) -> None`: Each thread stores
```